eng/pipelines/templates/jobs/promote-nuget-client.yml

```diff
@@ -0,0 +1,48 @@
+# NuGet.Client does not flow to the VMR directly but only to dotnet/sdk where it is validated
+# Upon the version bump happens in dotnet/sdk, this job promotes that build of NuGet.Client to
+# another channel from where it flows into the VMR.
+# The sdk and nuget.client subscriptions are batched and go into the VMR in a single PR.
+
+jobs:
```

Comment:
I would templatize this to accept a list of dependencies for which you want to promote a build rather than making this nuget specifc. The powershell script below would loop through those and promote each.

Reply:
Let's only do that once we have the need. I parametrized the name but kept a single one for now.
It might even be better to just clone more jobs if you want multiple packages/builds promoted because the error handling and auditing might be just easier.